### ISSUE TITLE
FT power too high

### ISSUE THREAD

**Issue by Video90** [Sublimis/UrbanBiker#88]
Hi, 
Great app, love it!
But: this new feature FT power is off. It is much too high (bike). Last days I did a tour of 1 hour, 2 times, nearly max speed. Average power both times about 215-220 Watts, which is correct for me. FT power, which is prominently visible in the summary-screen, states 310 Watts (first time) and 280 Watts (second time). 
(Noteworthy: the wind was blowing quite hard, especially on the first run. With tailwind in the first half of the route my speed was high. Second half there was headwind with low speed)
Please change algoritm, or at least put back average power in summary screen.

**Comment by Sublimis** [comment]
Hi, would it be possible to send us those two rides, e.g. by email? Most importantly, summary files and active power graphs are what would be great to examine before we can make final verdict about this (GPX is not needed, for privacy concerns).

Was the power sensor used? Also, what are the average _active_ power values?

The thing is, these FTP figures do not seem way off. Please note they're just estimates, which is especially true when no power sensor was used, and there was wind blowing. 220 W average power (AVP) is quite respectable, and FTP can't be lower than this. So in fact, FTP only 25% higher than AVP looks reasonable given all the estimates. AVP takes a whole ride into account, even when you're decelerating or going downhill, whereas our FTP algorithm, on the other hand, takes only strenuous parts of a ride into account. It's quite simple, basically has only one parameter to adjust. We could try adjusting it a bit to make it less sensitive, but it would be best to make some additional research first.

Until we implement customization of prominent values in summary screen, you can hide the FTP value by taping the field (so only the top three values are visible).

Please let us know here, or drop us an email. Thanks for the feedback!

**Comment by Video90** [comment]
Hi, what email adress? With contact@urban-bike-computer.com I get error messages.
Also: I know for sure that my max power which I can hold for 3-5 minutes is about 280-300 Watts, depending on my level of training. I assume FTP is the power which you can hold for longer periods, at least 20 minutes or so?

**Comment by Sublimis** [comment]
Yes, this is the address and we did get your emails, thanks for that. What do the errors say?

After examining the tracks you've sent us, it seems like you didn't adjust the Power parameters at all. The values are all set to their defaults, although they are crucial for getting accurate power readings without a power meter.

1. Weight values you use are _75 kg_ for the rider (you) and _15 kg_ for the bicycle, and no wheels weight set up. Chances are high that these values don't match your situation.
2. Drag area value (_Cd*A_) you use is 0.5 (the default), although more appropriate value would be ~ _0.4_ for MTB, or ~ _0.3_ for road.

Please adjust these parameters to get more accurate power readings. The easiest way is to open the Control Panel and click on the **Power/Energy parameters** section. The defaults are there just so a person can get a rough sense of power and energy spent, they are not meant to be taken for granted. Thanks for the feedback!

**Comment by Video90** [comment]
Hi,

Error was: 
A message that you sent could not be delivered to one or more of its
recipients. This is a permanent error. The following address(es) failed:

​ ​ sublimis@gmail.com
​ ​ ​ ​ (ultimately generated from contact@urban-bike-computer.com)

**Comment by Video90** [comment]
Regarding the parameters: my body and bike weight are correctly mentioned in my app-settings: 80 kg and 10 kg.  I adjusted my frontal area to 0,35 and set the wheels to 2kg (had never seen these values, thanx for the suggestion).

But.....this does not explain the strangely high Ftp's, especially in comparison with average power. 2 days ago I did another ride, same route, but this time witn little wind. Also same effort. As can be expected with little wind my average speed was a little higher. But the Ftp was lower, now 257 Watts. So in this algoritm my Ftp depends on the force of the wind: how stronger the wind, how higher the Ftp, with large differences of 50 Watts . This ofcourse cannot be true. It seems the algoritm looks at segments in which the speed is relatively high and calculates the Ftp from those segments. But  the algoritm has no way of knowing whether the speed is intermittent high because I give all my energy (interval training)   ór because I have strong tailwind, right? So when the user has no power sensor, the app cannot calculate Ftp reliably. 
Again I would like you to put the average power in the summary screen, I always look at that number to compare my ride with previous rides. The Ftp has no relevance, at least not in my situation.
